### ISSUE TITLE
ssh-add fails with "Error Loading Key: error in libcrypto"

### ISSUE THREAD

**Issue by Mara-Li** [webfactory/ssh-agent#97]
![image](https://user-images.githubusercontent.com/30244939/133517478-1dd4b6ff-d0a0-4d33-98ec-114299246429.png)
I have this error when trying to clone submodule (private).
I add the public key on the submodule without the comment ; And I add the private key on the public repo with link to the private repo.

**Comment by mpdude** [comment]
Is this still an active issue?

Seems to be an error message generated by `ssh-add` when trying to load a key.

https://github.com/openssl/openssl/issues/13443 might be related.

**Comment by xkomiks** [comment]
Relevant

**Comment by vokshirg** [comment]
> Relevant

It helped me https://github.com/openssl/openssl/issues/13443#issuecomment-1354750805

**Comment by ibnahzab** [comment]
TLDR; so basically You just add newline to make it works, thanks.
